watertap/unit_models/cstr.py

```diff
@@ -0,0 +1,84 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+Anoxic CSTR unit model for BSM2 and plant-wide wastewater treatment modeling.
```

Comment:
Rename filename to `anoxic_cstr` instead of `cstr`. Either that, or rename the class which is currently named `AnoxicCSTR`. The only dilemma I see with this sort of naming is that anoxic is not anaerobic, technically. So maybe it makes more sense to rename the class as CSTR, which would just build on top of IDAES' CSTR and be the WaterTAP version,